libs/juno-ui-components/README.md

```diff
@@ -17,8 +23,27 @@ in your app
 yarn install
 ```
 
+## Working With Tailwind
+Juno comes with [Tailwind](https://tailwindcss.com/) included, so when using Juno you automatically can use tailwind in your project.
+
+## Color Variables
+The Juno Design System comes with all colors defined as CSS variables ("custom properties").
+Juno includes SAP brand colors (`--color-sap-…`), Juno specific colors (`--color-juno-…`), Juno theme specific colors (`--color-juno-theme-…`) as well as semantic colors.
+
```

Comment:
I'm not sure we should mention the CSS variables here since we don't really want people to use those. They should use the tailwind classnames instead. At least as long as this document is meant as a quick "getting started" for consumers of our lib.